src/turtle.rs

```diff
@@ -199,6 +199,46 @@ impl Turtle {
         block_on(self.turtle.wait(secs))
     }
 
+    /// Instructs the turtle to draw an arc by the given angle for a given radius in the given step count from the current position of the turtle.
+    ///
+    /// The `radius` parameter is a floation point number that represents the radius of the arc
+    /// that you want to draw. If a negative `radius` is passed its absolute value is
+    /// considered for drawing the arc.
```

Comment:
```suggestion
    /// The `radius` parameter represents the radius of the arc that you want to draw.
    /// If a negative `radius` is passed its absolute value is considered for drawing the arc.
```

Fixed a typo and rewording a bit. 

Are we sure that we should be taking the absolute value of the radius? The [Python `circle` function](https://docs.python.org/3.3/library/turtle.html?highlight=turtle#turtle.circle) actually allows for radius to be negative:

> Draw the arc in counterclockwise direction if radius is positive, otherwise in clockwise direction.